cpp/src/utilities/error_utils.hpp

```diff
@@ -52,8 +65,8 @@ struct cuda_error : public std::runtime_error {
 #define CUDF_STRINGIFY(x) STRINGIFY_DETAIL(x)
 
 /**---------------------------------------------------------------------------*
- * @brief Error checking macro that throws an exception when a condition is
- * violated.
+ * @brief Macro for checking (pre-)conditions that throws an exception when  
```

Comment:
It's not just for pre-conditions.

Reply:
@jrhemstad : Which is why I put that into parentheses. Do you want me to remove that word entirely?

Reply:
Also, [this](https://github.com/rapidsai/cudf/pull/1386) should be merged first.

Reply:
This PR duplicates everything in #1386. I assume that was an accident.